packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -66,29 +66,28 @@ const InternalHeaderCell = <T extends TableItem>({
     );
   };
 
-  // TODO: Update this. Return early if we don't have tooltip.
   const renderTooltip = () => {
-    if (typeof tooltip === 'string' && tooltip.length > 0) {
-      return (
-        <Tooltip
-          id={tooltipId}
-          placement="right"
-          trigger={
-            <Box as="span" marginLeft="xxSmall">
-              <BaselineHelpIcon
-                aria-describedby={tooltipId}
-                size="medium"
-                title="Hover or focus for additional context."
-              />
-            </Box>
-          }
-        >
-          {tooltip}
-        </Tooltip>
-      );
+    if (!(typeof tooltip === 'string' && tooltip.length > 0)) {
```

Comment:
🍹 💅 Can we spread the negation or return it back to the original `if { return () }; return null;` route?
```suggestion
    if (typeof tooltip !== 'string' || !tooltip.length)) {
```

Reply:
Yes,  I have returned it back to its original state.